<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>586</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Small Video Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="srcBtn">
        <property name="text">
         <string>Source</string>
        </property>
        <property name="icon">
         <iconset resource="pictures.qrc">
          <normaloff>:/icons/images/folder_16x16.png</normaloff>:/icons/images/folder_16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sourcePath">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioBtn1">
        <property name="text">
         <string>MP4</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn2">
        <property name="text">
         <string>DV</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn3">
        <property name="text">
         <string>MOV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn4">
        <property name="text">
         <string>Audio</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="goBtn">
        <property name="text">
         <string>Go!</string>
        </property>
        <property name="icon">
         <iconset resource="pictures.qrc">
          <normaloff>:/icons/images/ok_16x16.png</normaloff>:/icons/images/ok_16x16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelVideo">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Video&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="labelResolution">
                  <property name="text">
                   <string>Resolution</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboResolution">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="labelAspect">
                  <property name="text">
                   <string>Aspect</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboAspect">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="labelCodec">
                  <property name="text">
                   <string>Codec</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboCodec">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="labelBitrate">
                  <property name="text">
                   <string>Bitrate</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBitrate">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>38</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="labelAudio">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Audio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="labelSampleRate">
                  <property name="text">
                   <string>Sample Rate</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboSamplerate">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="labelChannel">
                  <property name="text">
                   <string>Channel</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboAChannel">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="labelACodec">
                  <property name="text">
                   <string>Codec</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboACodec">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="labelABitrate">
                  <property name="text">
                   <string>Bitrate</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="margin">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboABitrate">
                  <property name="minimumSize">
                   <size>
                    <width>115</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>78</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="destBtn">
              <property name="text">
               <string>Destination</string>
              </property>
              <property name="icon">
               <iconset resource="pictures.qrc">
                <normaloff>:/icons/images/folder_16x16.png</normaloff>:/icons/images/folder_16x16.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="destPath">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIndicator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Indicator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>518</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="pictures.qrc">
          <normaloff>:/icons/images/cancel_16x16.png</normaloff>:/icons/images/cancel_16x16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="pictures.qrc">
          <normaloff>:/icons/images/door_16x16.png</normaloff>:/icons/images/door_16x16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="pictures.qrc">
     <normaloff>:/icons/images/exit_16x16.png</normaloff>:/icons/images/exit_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="pictures.qrc">
     <normaloff>:/icons/images/about_16x16.png</normaloff>:/icons/images/about_16x16.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>887</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>executeCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtn4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMP3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtn2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_processCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtn1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMP4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioBtn3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMOV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>executeCommand()</slot>
  <slot>setSource()</slot>
  <slot>setDestination()</slot>
  <slot>setMP3()</slot>
  <slot>setMP4()</slot>
  <slot>on_processCancel()</slot>
  <slot>setDV()</slot>
  <slot>setMOV()</slot>
 </slots>
</ui>
